src/webots/gui/WbSingleTaskApplication.cpp

```diff
@@ -231,10 +231,10 @@ void WbSingleTaskApplication::showSysInfo() const {
   // An OpenGL context is required there for the OpenGL calls like `glGetString`.
   // The format is QSurfaceFormat::defaultFormat() => OpenGL 3.3 defined in main.cpp.
   QOpenGLContext *context = new QOpenGLContext();
-  context->create();
+  assert(context->create());
```

Comment:
This will get optimized out in a release build. I'm actually not sure why it's not breaking the CI now :/
```suggestion
  if(!context->create())
    assert(false);
```